functions/src/repository/info.ts

```diff
@@ -111,19 +111,14 @@ export const getRepositoryInfo: any = async (token: string, repository: { uid: s
     .doc(mappedData.uid)
     .set(mappedData, { merge: true });
 
-  return mappedData;
-};
+  // Saving the Pull Request status and build time data
+  const promises: Promise<any>[] = [];
+  mappedData.pullRequests.map((pullRequest: GitHubPullRequestModel) => {
+    const ref: string = pullRequest.statusesUrl.split('/').pop();
+    promises.push(getPullRequestStatus(token, repository.fullName, ref, mappedData.uid, pullRequest.uid))
```

Comment:
`getPullRequestStatus` does not return a promise? Used here `functions/src/repository/response-git-webhook-repository.ts:159` without a promise

Reply:
It should return `Promise<GitHubPullRequestStatusModel[]>`. It have added the return type here and ` Promise.resolve(getPullRequestStatus(githubToken, repository.fullName, ref, repository.uid, data.pull_request.id));` inside `functions/src/repository/response-git-webhook-repository.ts:159`

Please let me know in case of any changes required